File: problems/0455-assign-cookies/solution.py
class Solution(object):
    def findContentChildren(self, g, s):
        if(s==[] or g==[]):
            return 0
        s.sort()
        g.sort()
        c=0
        i=0
        j=0
        while((i<len(g)) and (j<len(s))):
            if(s[j]>=g[i]):
                c+=1
                i+=1
            j+=1       
        return c           
        

File: problems/0455-assign-cookies/solution.java
class Solution {
    public int findContentChildren(int[] g, int[] s) {
        if(s.length==0 || g.length==0){
            return 0;
        }    
        Arrays.sort(s);
        Arrays.sort(g);
        int c=0,i=0,j=0;
        while(i<g.length && j<s.length){
            if(s[j]>=g[i]){
                c++;
                i++;
            }  
            j++;
        }          
        return c; 
    }
}
